src/protagonist/DLCS.AWS/Cloudfront/CacheInvalidator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Amazon.CloudFront;
+using Amazon.CloudFront.Model;
+using DLCS.AWS.Settings;
+using DLCS.Core;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace DLCS.AWS.Cloudfront;
+
+public class CacheInvalidator : ICacheInvalidator
```

Comment:
Small thing, is it worth naming this `CloudfrontInvalidator`, in the future we may support other CDNs (e.g. `FastlyInvalidator : ICacheInvalidator`)